test/components/structures/TimelinePanel-test.tsx

```diff
@@ -259,6 +259,39 @@ describe("TimelinePanel", () => {
                 setUpTimelineSet();
             });
 
+            describe("and receving an event", () => {
+                beforeEach(async () => {
+                    await renderTimelinePanel();
+                    timelineSet.addLiveEvent(ev1, {});
+                    await flushPromises();
+                    // @ts-ignore Simulate user activity by calling updateReadMarker on the TimelinePanel.
```

Comment:
Did you intend these `@ts-ignore`s? If so, why do you need them?

Reply:
I blatantly copied these lines from another test.

Reply:
It's because they are calling private methods.

Reply:
Ah, then can you just write `// @ts-ignore calling private method` to make it clear? Thanks.

Reply:
Yes, that's what I did.